yarn.lock

```diff
@@ -1480,9 +1480,9 @@
   integrity sha512-6U71C2Wp7r5XtFtQzYrW5iKFT67OixrSxjI4MptCHzdSVlgabczzqLe0ZSgnub/5Kp4hSbpDB1tMytZY9pwxxA==
 
 "@emurgo/cardano-serialization-lib-nodejs@^9.1.0":
```

Comment:
I see this update in the `serialization-lib` used for the tests, but I don't see the update in the `package.json`, you can add the `package.json` or drop this file.